tcpdf.php

```diff
@@ -18922,13 +18922,26 @@ protected function openHTMLTagHandler($dom, $key, $cell) {
 				$prevlinewidth = $this->GetLineWidth();
 				$this->SetLineWidth($hrHeight);
 
-				$lineStyle = array(
-					'color' => $tag['fgcolor'],
-					'cap'   => $tag['style']['cap'],
-					'join'  => $tag['style']['join'],
-					'dash'  => $tag['style']['dash'],
-					'phase' => $tag['style']['phase'],
-				);
+				$lineStyle = array();
+
+				if (isset($tag['fgcolor'])) {
+					$lineStyle['color'] = $tag['fgcolor'];
+				}
+
+				if (isset($tag['style']) && is_array($tag['style'])) {
```

Comment:
```suggestion
				if (isset($tag['style'])) {
```

Reply:
Where did you see it is an array for sure ?

Reply:
Where did you see non-array input is better to be silently accepted/discarded?

Reply:
> Where did you see non-array input is better to be silently accepted/discarded?

Nowhere
php errors should be avoided, warning the user about a bad input is another problem 🤷🏻‍♂️

Reply:
`Undefined array key` etc. is exactly that warning, we do not use strict types here...

Reply:
After having a look at the code, I think we can safely assume the `style` key is either missing or an array. Search for `$dom[$key]['style']` in this file.